src/garnet/helpers/settings.py

```diff
@@ -0,0 +1,39 @@
+"""Global settings for Garnet."""
+from contextlib import contextmanager
+
+from mantid.kernel import ConfigService
+
+debug_init = False
+
+
+def init_debug() -> None:
+    """Initialize the debug flag."""
+    global debug_init  # noqa
+    if not debug_init:
+        debug_init = True
+        global debug_garnet  # noqa
```

Comment:
Rather than `global` variables, how about having this do
```python
mantid.ConfigService.setLogLevel(7)
```
then everything is handled internally in mantid